metadata-models/src/main/pegasus/com/linkedin/dataJob/azkaban/AzkabanFlowInfo.pdl

```diff
@@ -0,0 +1,17 @@
+namespace com.linkedin.dataJob.azkaban
+
+/**
+ * Metadata associated with a Azkaban flow
+ */
+record AzkabanFlowInfo {
+
+  /**
+   * Azkaban project information including project name, version and cluster
+   */
+  project: AzkabanProjectInfo
```

Comment:
Azkaban project sounds like yet another entity. I noticed that you didn't make it an entity (maybe just for now?).

Reply:
We don't have a use case/need till now where we need it to be an entity. We might do that in the future.